install/prepare_site_init.md

```diff
@@ -1,10 +1,10 @@
-# Prepare Site Init
+# Prepare `Site Init`
```

Comment:
If we're adding backticks we should correct the spelling.

Should be 

```markdown
# Prepare `site-init`
```

Reply:
done